src/decisionengine/framework/modules/tests/test_QueueLogger.py

```diff
@@ -0,0 +1,61 @@
+import gc
+import logging
+
+import pytest
+
+import decisionengine.framework.modules.QueueLogger as QueueLogger
+
+
+@pytest.fixture()
+def queue_logger_setup():
+    q_log = QueueLogger.QueueLogger()
+    yield q_log
+
+    q_log = None
+
+
+@pytest.fixture()
+def log_setup():
+    test_log = logging.getLogger()
+    yield test_log
+
+    gc.collect()
+
+
+@pytest.fixture()
+def handler_setup():
+    test_handler = [logging.StreamHandler()]
+    yield test_handler
+
+    gc.collect()
+
+
+@pytest.mark.usefixtures("queue_logger_setup", "log_setup", "handler_setup")
+def test_setup_queue_logging(queue_logger_setup, log_setup, handler_setup):
+    queue_logger_setup.setup_queue_logging(log_setup, handler_setup)
+
+    assert queue_logger_setup.structlog_q is not None
+    assert isinstance(queue_logger_setup.structlog_q_handler, logging.handlers.QueueHandler)
+    assert "QueueHandler" in str(log_setup.handlers)
+    assert isinstance(queue_logger_setup.structlog_listener, logging.handlers.QueueListener)
+
+
+pytest.mark.usefixtures("queue_logger_setup", "log_setup", "handler_setup")
```

Comment:
I believe this is missing the `@` (and the one below too)

Reply:
oops. fixing

Reply:
fixed